tools/Dockerfile

```diff
@@ -3,7 +3,7 @@ MAINTAINER "Stef Walter" <stefw@redhat.com>
 
```

Comment:
fedora:22 doesn't exist yet ... and given the wild state of Fedora, God knows what would get pushed in there ... lets stick with what we know ... and only update when after smoke testing a new base image.